src/Gui/Application.cpp

```diff
@@ -2005,49 +2005,50 @@ void Application::runApplication()
 #endif
 
     Application app(true);
-    MainWindow mw;
+    MainWindow mw {};
     mw.setProperty("QuitOnClosed", true);
 
     // allow to disable version number
     ParameterGrp::handle hGen = App::GetApplication().GetParameterGroupByPath("User parameter:BaseApp/Preferences/General");
-    bool showVersion = hGen->GetBool("ShowVersionInTitle",true);
+    bool showVersion = hGen->GetBool("ShowVersionInTitle", true);
 
     if (showVersion) {
         // set main window title with FreeCAD Version
-        std::map<std::string, std::string>& config = App::Application::Config();
-        QString major  = QString::fromLatin1(config["BuildVersionMajor"].c_str());
-        QString minor  = QString::fromLatin1(config["BuildVersionMinor"].c_str());
+        std::map<std::string, std::string> &config = App::Application::Config();
+        QString major = QString::fromLatin1(config["BuildVersionMajor"].c_str());
+        QString minor = QString::fromLatin1(config["BuildVersionMinor"].c_str());
         QString title = QString::fromLatin1("%1 %2.%3").arg(mainApp.applicationName(), major, minor);
         mw.setWindowTitle(title);
-    } else {
+    }
+    else {
         mw.setWindowTitle(mainApp.applicationName());
     }
 
     QObject::connect(&mainApp, SIGNAL(messageReceived(const QList<QByteArray> &)),
                      &mw, SLOT(processMessages(const QList<QByteArray> &)));
 
     ParameterGrp::handle hDocGrp = WindowParameter::getDefaultParameter()->GetGroup("Document");
-    int timeout = hDocGrp->GetInt("AutoSaveTimeout", 15); // 15 min
+    int timeout = hDocGrp->GetInt("AutoSaveTimeout", 15);// 15 min
```

Comment:
This particular formatting change seems like noise to me. I'd omit it from future PRs, it just adds review overhead.